guides/upgrading/index.md

```diff
@@ -1,39 +1,27 @@
-When someone says they are "upgrading" their Ember app, it could mean one of several things, especially if they say that they are "upgrading to Octane."
-This Guide will help fill in the blanks about how to upgrade your app's version, get access to the latest and greatest features, and form a strategy for using Octane's features in your existing apps.
+Quand des devs disent qu'ils "mettent à jour" leur app Ember, ça peut vouloir dire plusieurs choses, en particulier s'ils parlent de "passer à Octane". Cette page du Guide vous aidera à remplir les blancs sur ce qui concerne la mise à jour la version de votre app, l'accès aux dernières et meilleures fonctionnalités, et à définir une stratégie pour utiliser les fonctionnalités d'Octane dans vos apps existantes.
 
-## Details about recent releases
+## Détail des _releases_ récentes
 
-You can learn about what changed in every release by reading the 
-[official Ember blog](https://blog.emberjs.com).
+Vous apprendrez ce qui a changé dans chaque _release_ (version livrée) en lisant [le blog officiel d'Ember](https://blog.emberjs.com).
 
-## Routine minor version maintenance upgrades
+## Mises à jour de routine pour maintenir les versions mineures
 
-Let's say you are upgrading an app from `v3.4` to `v3.8`.
-Although you heard there are some new features, your main goal is to keep up with security updates.
-To do this kind of version upgrade, follow the instructions [here in the CLI Guides](https://cli.emberjs.com/release/basic-use/upgrading/).
-That process will make sure that any peer dependencies of the `ember.js` core codebase get upgraded too.
-Automated tools will help you make the right changes to `package.json` and other files.
-The point of a minor version bump is that you shouldn't need to change anything in your app when you upgrade - it should all keep working, whether or not you choose to adopt new syntaxes and features.
+Mettons que vous voulez mettre à jour une app de `v3.4` à `v3.8`. Bien que vous ayez entendu parler de nouvelles fonctionnalités, votre objectif principal est de suivre les mises à jour de sécurité. Pour faire ce type de mise à jour, suivez les instructions [ici, dans les Guides CLI](https://cli.emberjs.com/release/basic-use/upgrading/). Ce processus vous assurera que toutes les dépendances de la base de code `ember.js` sont également mises à jour. Des outils automatisés vous aideront à faire les bons changements dans le `package.json` et d'autres fichiers. L'intérêt d'un saut de version mineure est que vous ne devriez pas avoir besoin de changer quoi que ce soit dans votre application lors de la mise à jour&nbsp;; tout devrait continuer à fonctionner, que vous choisissiez ou non d'adopter de nouvelles syntaxes et fonctionnalités.
```

Comment:
```suggestion
Mettons que vous voulez mettre à jour une app de `v3.4` à `v3.8`. Bien que vous ayez entendu parler de nouvelles fonctionnalités, votre objectif principal est de suivre les mises à jour de sécurité. Pour faire ce type de mise à jour, suivez les instructions [ici, dans le Guide CLI](https://cli.emberjs.com/release/basic-use/upgrading/). Ce processus vous assurera que toutes les dépendances de la base de code `ember.js` sont également mises à jour. Des outils automatisés vous aideront à faire les bons changements dans le `package.json` et d'autres fichiers. L'intérêt d'un saut de version mineure est que vous ne devriez pas avoir besoin de changer quoi que ce soit dans votre application lors de la mise à jour&nbsp;; tout devrait continuer à fonctionner, que vous choisissiez ou non d'adopter de nouvelles syntaxes et fonctionnalités.
```

Nitpicking: I would keep the same line breaks at it makes the reading easier, less compact.

Reply:
To be honest their pseudo-line breaks that are not proper line-breaks get me mad and make my reading harder. I prefer the markdown to represent clearly what I see on screen. It's a question of preference really.